functests/customResource_test.go

```diff
@@ -0,0 +1,125 @@
+package functests
```

Comment:
Rename the file to custom_resource_test.go

Reply:
Or better cincinnati_creation_test.go?

Reply:
ok

Reply:
Done